docs/user-guide/api-mediation/discovery-service-configuration.md

```diff
@@ -37,6 +37,23 @@ As an application developer who wants to run Zowe, set the following parameters
             discovery:
                 staticApiDefinitionsDirectories: config/local/api-defs;config/local2/api-defs
 
+* **apiml.discovery.serviceIdPrefixReplacer**
+    
+    This parameter is used to modify the service ID of a service instance, before it registers to API ML.
+    Using this parameter ensures compatibility of non-conformant services with v2, based on Zowe v2 conformance.
+    The value of the `*apiml.discovery.serviceIdPrefixReplacer` parameter is represented as a tuple that contains two strings, separated by a comma. The format of this parameter contains the following two elements:
+    * First, the prefix that you want to replace in the service ID
+    * Second, the new prefix that will be replaced
+    
+  **Example:**
+    The value of the parameter has the following format: `oldServiceIdPrefix,newServiceIdPrefix`
+
+    Set this parameter in the Zowe configuration by specifying the parameter `ZWE_components_discovery_server_prefix_replacer`.
+    The parameter value is then propagated to the Discovery Service. 
```

Comment:
We can rephrase to something like this, since we don't encourage end-user to define environment variables directly.

Set this parameter in you Zowe YAML configuration (typically zowe.yaml) by defining `components.discovery.server.prefix.replacer`. For example, defining it globally:

```yaml
components:
  discovery:
    server:
      prefix:
        replacer: caservice,bcmservice
```

Or defining it only for `lpar1` high availability instance:

```yaml
haInstances:
  lpar1:
    components:
      discovery:
        server:
          prefix:
            replacer: caservice,bcmservice
```

(maybe we shouldn't suggest end-user to define different values on each HA instance in this case, then we should remove the section sample.)

Reply:
thanks jack, I will apply the changes